filesystem/processor.go

```diff
@@ -46,7 +46,7 @@ func newProcessor(options options) *processor {
 func (p *processor) process(source, destination string) error {
 	si, err := os.Stat(source)
 	if err != nil {
-		logrus.Errorf("Unable to stat %s : %s", source, err)
+		logrus.Errorf("Unable to stat source [%s] in filesystem.process(): %s", source, err)
```

Comment:
Since this is a error message let's not expose function names here. Probably change it wo 'Unable to stat souce [%s] while processing in filesystem'

Reply:
done